repose-aggregator/components/filters/uri-user-filter/build.gradle

```diff
@@ -1,12 +1,15 @@
 dependencies {
     compile project(":repose-aggregator/core/core-service-api")
     compile project(":repose-aggregator/commons/commons-utilities")
-    compile project(":repose-aggregator/core/core-lib") //todo: remove this once the wrappers have been removed
     compile "org.slf4j:slf4j-api"
+    compile "org.scala-lang:scala-library"
+    compile "com.typesafe.scala-logging:scala-logging-slf4j_2.10"
 
     provided "javax.servlet:javax.servlet-api"
     provided "javax.inject:javax.inject"
 
     testCompile "junit:junit"
+    testCompile "org.scalatest:scalatest_2.10"
     testCompile "org.mockito:mockito-all"
+    testCompile "org.springframework:spring-test"
```

Comment:
No idea.  It's copy/pasta from another file that seemed to work.  I had not realized it didn't have the jcl-over-slf4j in there.

@adrianjgeorge ?